### PR TITLE
chore: Improve links readability 

### PR DESCRIPTION
**Pull request by yash-rajpal**
## Proposed changes

Moving the TnC links on login and sign up pages in new lines for better readability by on screen readers. 
### Before
image

### After 
image 
image

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

[MA-76](https://rocketchat.atlassian.net/browse/MA-76)

[MA-76]: https://rocketchat.atlassian.net/browse/MA-76?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by diegolmello — CHANGES_REQUESTED** [review]
Missing these changes

image

**Comment by yash-rajpal** [comment]
I saw these changes on [MA-74](https://rocketchat.atlassian.net/browse/MA-74) too, I was planning to ship these with the other task.

**Comment by diegolmello** [comment]
> I saw these changes on [MA-74](https://rocketchat.atlassian.net/browse/MA-74) too, I was planning to ship these with the other task.

Are you sure? I was not able to find them there.

**Comment by yash-rajpal** [comment]
Yes, it was in the designs for that other task too.

No worries, let me do these with this task itself.

File: app/views/LoginView/styles.ts [hunk 1]
```diff
@@ -21,16 +21,21 @@ export default StyleSheet.create({
...
 	},
 	ugcContainer: {
 		marginTop: 32
+	},
+	createAccountButton: {
+		width: '100%',
+		marginTop: 12
+	},
+	forgotPasswordButton: {
+		width: '100%',
```
Comment (diegolmello) [inline app/views/LoginView/styles.ts:38]: I'm pretty sure I mentioned on a previous review, but you should be using flexbox instead of width `100%`.

**Review by diegolmello — CHANGES_REQUESTED** [review]

**Review by diegolmello — CHANGES_REQUESTED** [review]
@yash-rajpal Hey Yash. Please take a look at the recent commits I did to display how some small changes affect the overall code readability, composability and UX :)

Please start using gap instead of margins. This is not present on the rest of the app, because it was introduced recently on a recent release of React Native. We're going to start migrating to this new way of composing components in baby steps, like this PR.

Also please make sure to check text fonts again. This is a accessibility project and small details matter 🤞🏼

File: app/views/RegisterView.tsx [hunk 2]
```diff
@@ -307,18 +306,16 @@ class RegisterView extends React.Component<IProps, any> {
 						style={styles.registerButton}
 					/>
 
-					<UGCRules />
-
 					{showLoginButton ? (
 						<View style={styles.bottomContainer}>
 							<Text style={[styles.bottomContainerText, { color: themes[theme].fontSecondaryInfo }]}>
```
Comment (diegolmello) [inline app/views/RegisterView.tsx:311]: Not matching Figma

File: app/containers/UserGeneratedContentRules.tsx [hunk 3]
```diff
@@ -15,14 +15,16 @@ const styles = StyleSheet.create({
...
 	bottomContainerText: {
-		...sharedStyles.textRegular,
-		fontSize: 13,
+		...sharedStyles.textMedium,
+		fontSize: 14,
+		lineHeight: 22,
 		textAlign: 'center'
 	},
-	bottomContainerTextBold: {
-		...sharedStyles.textSemibold,
```
Comment (diegolmello) [inline app/containers/UserGeneratedContentRules.tsx:23]: Not matching Figma.

You could do this
```
bottomContainerText: {
  ...sharedStyles.textMedium,
  fontSize: 14,
  lineHeight: 22,
  textAlign: 'center'
},
  bottomContainerTextUnderline: {
  textDecorationLine: 'underline'
}
```

```
style={[styles.bottomContainerText, styles.bottomContainerTextUnderline...
```

**Review by diegolmello — CHANGES_REQUESTED** [review]

File: app/containers/UserGeneratedContentRules.tsx (hunk 3 above)
Comment (yash-rajpal) [reply app/containers/UserGeneratedContentRules.tsx:23]: Oops, I didn't realize medium text. Done! 

Thanks!

File: app/views/RegisterView.tsx (hunk 2 above)
Comment (diegolmello) [reply app/views/RegisterView.tsx:311]: @yash-rajpal did you see this?

Comment (yash-rajpal) [reply app/views/RegisterView.tsx:311]: I thought it was related to inside this component 🤔 

Let me see.

Comment (yash-rajpal) [reply app/views/RegisterView.tsx:310]: Updated a few styles, please check. 😬

Comment (yash-rajpal) [reply app/views/RegisterView.tsx:310]: Also here - 
image
Not updating `App version` to `Version` as in styles as this change was recently made [here](https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5846) and the designs are older than that.

**Review by diegolmello — APPROVED** [review]

**Pull request merged**